---
title: Response to reviewer 1's comment, G3
author: Bin He
date: 7 fev 2018
categories:
  - manuscript
  - collaboration
tags:
  - pho
  - dennis wykoff
---

## Reviewer's comment

... the authors focus on adaptation but should meanwhile consider and discuss the effects of genetic drift in many cases. Additionally, are there any features implying genomic fragile sites or recombination hotpots in any species that might link to the repeated changes here?

## Notes

There are two questions here. The first concerns the evolutionary forces driving the gene family evolution and specialization. The second concerns the mutational processes behind the same pattern.

We will address the second question first, since it's the mutational processes that generate the raw variation upon which selection acts. First of all, it is worth pointing out that mutations, including duplications and promoter / protein changes, are random. What the reviewer's comment concerns then, is the frequency of the mutations compared to other similar situations, namely whether the evolutionary history for these two phosphatase families are exceptionally dynamic, and whether that is because of the genomic regions they reside in being especially prone to gene duplication and losses etc. 

While we were impressed by the frequent expansion and contraction of the _PHO5_ family based on our phylogenetic analysis, we didn't make direct comparison between this and other gene families. That said, the reviewer did raise an interesting possibility about the mutational processes behind the observed evolutionary pattern. For one thing, it is known that tandemly duplicated loci are prone to expansion and contraction due to misalignment during homologous recombination. This is more likely to happen when the duplication events were relatively recent so that the copies share high sequence identity. One plausible example of this is the _S. mikatae_ syntenic region with _ScPHO5_ and _ScPHO3_: it and no other _sensu stricto_ species has three instead of two copies, and the sequence similiarity between the paralogs is high (?%). 

We then examined published recombination hot spots in _S. cerevisiae_ S288C, asking if the _PHO5_ / _PHO3_ region is especially prone to recombination mediated gene copy number variation. However, we didn't find that to be the case. While similar data in other species are not available, other evidence suggests that homologous recombination mediated copy number changes unlikely play a major role. For example, our gene phylogeny reconstruction (Supplementary Figure 2) shows that four duplication events occurred in the lineage leading to 5 _PHO5_ homologs in _K. lactis_, and yet the five paralogs in the _K. lactis_ genome all reside on different chromosomes.

A similar question may be asked about the mutations behind the transcription and substrate specialization, namely whether genome fragility or local sequence composition may have made such changes more likely to happen than in other genes. To answer this question, however, requires the dissection of the sequence changes behind the promoter and protein functional differences, which we have made some progress (Orkwis, Orlando) but still need to get more data on before a satisfying answer can be provided.

For the first question, while we have focused on adaptive forces in the first paragraph of Discussion, we agree that genetic drift likely plays an important role in generating the evolutionary pattern of _PHO5_ family. --- Here is just me thinking aloud: one place that genetic drift may play a role is the loss of genes. Since there is no evidence suggesting that losing a phosphatase gene is beneficial, we may postulate that losses might have simply been tolerated, and fixed in the population by random processes, i.e. genetic drift. It is harder to imagine a neutral explanation for the preservation of gene duplicates and their specialization. However, it is not impossible. For example, some gene duplicates may have been fixed by genetic drift, just as gene losses were. Afterwards, for the duplicated genes to be preserved, sub- or neo-functionalization may be invoked. In those cases, splitting the ancestral function, or one copy acquiring a new function, may simply be necessary to have both (or more) copies being maintained. Michael Lynch, for example, is an advocate for neutral processes generating seemingly complex genomic architecture. If he were reviewer 1 (which I doubt, as he will probably be more harsh), this will be his position by default.

The bottom line is, I think we need to tone down the adaptive interpretation in the first paragraph of Discussion. 

Just for fun, I searched throughout the manuscript, and found that we used the term "adapt" five times, once in the Abstract, twice in the first paragraph of Introduction, and twice in the first paragraph of Discussion. Those in Introduction don't matter because it was not a direct statement on our work. The other three times were for the same message, namely the convergent evolution we observed in the _PHO5_ and _PMU_ families suggest common selective pressures for the specialization of the phosphatases against different nutrient starvation conditions. What's curious is the fact that such gene duplication and specialization have been accompanied by gene losses. If it is useful to have specialized phosphatases, why lose them and re-evolve them again? For this, we speculated that the selective pressure favoring the presence of multiple, specialized phosphatases is either not very strong, or its intensity changes over time and differs between species.

### Gene duplication

Reviewer 1 asked whether suggested is with respect to the frequency of such mutations. In other words, the question is whether the evolutionary gain and loss of the phosphatase encoding genes, and the subsequent evolution of their regulation and function, happen more often than other gene families, and if so, could it be because the ancestral _PHO5_ gene resided in a particular part of the chromosome that is especially prone to gene duplication and loss?
