---
title: G3 reviewers' comments, first round
author: Bin He
date: 2 fev 2019
---

## Reviewer 1

The authors demonstrated the phosphatase gene families in many Ascomycetes dynamically evolved through repeated gene duplication, functional specialization, and gene loss. Specifically, they showed the specialization of duplicate genes CgPMU2 and CgPMU3 at the level of transcription and substrate utilization. They identified CgPMU3 as a possible link to functionally compensate for the loss of ScPHO3 homolog in Candida glabrata (C. glabrata), in response to thiamine starvation. They showed that the recent duplicate gene pair, CgPMU2 and CgPMU3, have functionally diverged to primarily respond to phosphate starvation and thiamine starvation, respectively. Function divergence of duplicate genes in response to phosphate starvation or thiamine starvation appears to also occur in Saccharomyces cerevisiae and Schizosaccharomyces pombe. By allele swap, they further compared the enzyme specificities of these duplicate genes from these three species. The common amino acids shared between CgPMU2 and CgPMU3 appears to provide function similar to CgPMU3, in terms of the specificity to TPP. They nicely showed the dynamic evolution of these two gene families and I appreciate the functional data they performed. This work will benefit our understanding of the dynamic functional divergence of duplicate genes. 

### Major comments: 

There are several aspects should be addressed: 
First, the authors focus on adaptation but should meanwhile consider and discuss the effects of genetic drift in many cases. Additionally, are there any features implying genomic fragile sites or recombination hotpots in any species that might link to the repeated changes here? 
Second, I found that there is no statistical analysis throughout the paper and the number of experiment repetitions is not indicated. In the RNA-Seq data, apparently only one replicate was used for each genotype and condition, according to the description in the method and supplement. If this is the case, I discourage the usage of this dataset as a main figure (Figure 1) but instead as a supplementary figure. Additionally, qPCR is encouraged to support the data presented in Figure 1B if only one replicate is used. 
Third, the functional link between thiamine and phosphate metabolism demonstrated here is indirect. It sounds there should be some profound implication here. It is likely that the link is an intrinsic feature of how these two metabolic processes might couple, as discussed by the authors in Line 375-378. Nonetheless, this link might not be related to similar environmental pressure but likely merely a neutral process. 

### Minor comments: 

1. I find the functional link of thiamine and phosphate metabolism is not clearly developed in the text. It will be helpful to include a more detailed and clearer introduction of the pathways known to metabolize thiamine and phosphate, their starvation regulation, any known possible cross-talks, metabolic or regulatory links in the introduction section. A figure or diagram might help in this case. 
2. Please include legends in Figure S2. 
3. Since C. glabrata is a pathogen, in which gene loss is common, it will be important to consider and discuss the impacts of pathogenic lifestyle on the dynamic evolution. I suspect that being a pathogen is also an important factor of using a different set of genes instead of homologs of ScPHO3 and ScPHO5. 
4. The method section does not have the information of genetic engineering and transformation method used in C. glabrata. 
5. There is no error bar in Figure 4. 
6. L140-141: "some potential cross-talk between the transcription factors", it could be cross-talks between metabolism per se instead of at the regulation level; "an artifact of the plasmid reporter", what possible artifacts are the authors referring here? 
7. L286: "remove native PNPPase", please include explanation or reference to support that the PNPPase activity in C. glabrata is mainly provided by PMU2. 
8. L303: "different structures in the active site", is there any reference to support this claim? 
9. L394-404: this paragraph is too speculative. 


## Reviewer \#2

In general, this article is good and well written. In this study, the authors performed RNA-seq on C. glabrata cells and identified CgPMU3 as a highly induced gene during thiamine starvation. They further determined that both CgPMU3 and CgPMU2 are capable of cleaving the phosphatase substrate, 1-napthyl-phosphate and members of this family evolved novel enzymatic functions for phosphate and thiamine starvation. Moreover CgPMU3 and CgPMU2 were suggested to be regulated transcriptionally by nutrient conditions. 

My comments regarding the article are written below, 

- Introduction, Page 2: While the authors introduced different fates of duplicate genes, they should also write briefly write about gene loss followed by gene duplication, referring to the previous studies (Lynch and Conery 2000, Naseeb et al 2017, Scannell et al 2006) 

- Figure 1 B: Is the fold change shown for thi3Δ significant? Why no observations were taken for thi3Δ after 2 hours? 

- Figure 2: Does the induction depend amount the concentration of thiamine in the media and if it varies with different concentrations of thiamine and phosphate? 

- Figure 4: The graph should be with error bars. 

- Page 10, lines 223-231: Did the authors look at the expression of PHO2 gene? Since PHO2 binds to PHO5 promoter, hence its expression can also be changed under thiamine and phosphate depletion. 

- Page 16, line 358: What phenotype is observed by inserting the PHO5 gene in C. glabrata? 

- Page 18, line 412: Was the RNA isolated at both 2 and 4 hours? 

- Pgae 18, line 414: Please be more clear which six samples were sequenced?
