<html><head>
<meta http-equiv="content-type" content="text/html; charset=windows-1252"><title> SignalP 4.1 Server - prediction results</title>
</head><body vlink="#FF3399" text="#000000" link="#FF3399" bgcolor="#f8f8f8" alink="#808080">
<font face="ARIAL,HELVETICA">
<br>
<table>
<tbody><tr><td><img src="PMU1_homologs_Ascomycetes_20171108_files/m_logo.gif">
    </td><td>&nbsp;&nbsp;&nbsp;
    </td><td><h2>SignalP 4.1 Server - prediction results</h2>
        <h3>Technical University of Denmark</h3>
</td></tr></tbody></table>
<br>
</font>
<hr>
<pre><pre># SignalP-4.1 euk predictions
<b>&gt;Scer_YKL128C 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Scer_YKL128C.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    18       0.123
  max. Y    18       0.117
  max. S     1       0.141
  mean S     1-17    0.103
       D     1-17    0.109   0.450   NO
Name=Scer_YKL128C	SP='NO' D=0.109 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Scer_YKL128C.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Scer_YKL128C.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Spar_spar321-g33.1 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Spar_spar321-g33.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.121
  max. Y    18       0.121
  max. S     1       0.152
  mean S     1-17    0.119
       D     1-17    0.120   0.450   NO
Name=Spar_spar321-g33.1	SP='NO' D=0.120 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Spar_spar321-g33.1.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Spar_spar321-g33.1.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Smik_smik702-g22.1 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Smik_smik702-g22.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.118
  max. Y    11       0.111
  max. S     1       0.138
  mean S     1-10    0.100
       D     1-10    0.105   0.450   NO
Name=Smik_smik702-g22.1	SP='NO' D=0.105 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Smik_smik702-g22.1.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Smik_smik702-g22.1.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Sbay_sbayc559-g5.1 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Sbay_sbayc559-g5.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.123
  max. Y    11       0.116
  max. S     1       0.148
  mean S     1-10    0.107
       D     1-10    0.111   0.450   NO
Name=Sbay_sbayc559-g5.1	SP='NO' D=0.111 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Sbay_sbayc559-g5.1.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Sbay_sbayc559-g5.1.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cgla_CAGL0K07568g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cgla_CAGL0K07568g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    21       0.212
  max. Y    21       0.349
  max. S     5       0.747
  mean S     1-20    0.562
       D     1-20    0.464   0.450   YES
Name=Cgla_CAGL0K07568g	SP='YES' Cleavage site between pos. 20 and 21: NTA-QD D=0.464 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgla_CAGL0K07568g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgla_CAGL0K07568g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cgla_CAGL0K07546g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cgla_CAGL0K07546g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    21       0.168
  max. Y    21       0.301
  max. S     4       0.726
  mean S     1-20    0.527
       D     1-20    0.423   0.450   NO
Name=Cgla_CAGL0K07546g	SP='NO' D=0.423 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgla_CAGL0K07546g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgla_CAGL0K07546g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cgla_CAGL0K07524g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cgla_CAGL0K07524g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    21       0.138
  max. Y    21       0.289
  max. S     4       0.757
  mean S     1-20    0.594
       D     1-20    0.454   0.450   YES
Name=Cgla_CAGL0K07524g	SP='YES' Cleavage site between pos. 20 and 21: VEM-TV D=0.454 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgla_CAGL0K07524g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgla_CAGL0K07524g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Nbac_NABA0S25 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Nbac_NABA0S25.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    42       0.108
  max. Y    42       0.105
  max. S    27       0.111
  mean S     1-41    0.099
       D     1-41    0.102   0.450   NO
Name=Nbac_NABA0S25	SP='NO' D=0.102 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Nbac_NABA0S25.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Nbac_NABA0S25.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Nbra_CABR0S29 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Nbra_CABR0S29.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    28       0.115
  max. Y    28       0.110
  max. S    14       0.122
  mean S     1-27    0.105
       D     1-27    0.107   0.450   NO
Name=Nbra_CABR0S29	SP='NO' D=0.107 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Nbra_CABR0S29.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Nbra_CABR0S29.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ndel_NADE0S12 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ndel_NADE0S12.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    28       0.127
  max. Y    28       0.130
  max. S     7       0.185
  mean S     1-27    0.134
       D     1-27    0.132   0.450   NO
Name=Ndel_NADE0S12	SP='NO' D=0.132 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ndel_NADE0S12.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ndel_NADE0S12.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Nniv_CANI0S04 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Nniv_CANI0S04.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    28       0.139
  max. Y    28       0.129
  max. S     7       0.164
  mean S     1-27    0.121
       D     1-27    0.125   0.450   NO
Name=Nniv_CANI0S04	SP='NO' D=0.125 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Nniv_CANI0S04.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Nniv_CANI0S04.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Scas_Scas721.121 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Scas_Scas721.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    68       0.110
  max. Y    68       0.105
  max. S     4       0.110
  mean S     1-67    0.097
       D     1-67    0.101   0.450   NO
Name=Scas_Scas721.121	SP='NO' D=0.101 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Scas_Scas721.121.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Scas_Scas721.121.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Klac_KLLA0B12628g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Klac_KLLA0B12628g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.116
  max. Y    24       0.104
  max. S     2       0.118
  mean S     1-23    0.094
       D     1-23    0.099   0.450   NO
Name=Klac_KLLA0B12628g	SP='NO' D=0.099 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Klac_KLLA0B12628g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Klac_KLLA0B12628g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Sklu_SAKL0E03718g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Sklu_SAKL0E03718g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.132
  max. Y    24       0.115
  max. S    23       0.123
  mean S     1-23    0.101
       D     1-23    0.108   0.450   NO
Name=Sklu_SAKL0E03718g	SP='NO' D=0.108 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Sklu_SAKL0E03718g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Sklu_SAKL0E03718g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Agos_AEL304C 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Agos_AEL304C.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    19       0.468
  max. Y    19       0.550
  max. S    18       0.819
  mean S     1-18    0.628
       D     1-18    0.592   0.450   YES
Name=Agos_AEL304C	SP='YES' Cleavage site between pos. 18 and 19: ALA-AQ D=0.592 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Agos_AEL304C.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Agos_AEL304C.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Clus_CLUG01309 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Clus_CLUG01309.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    17       0.149
  max. Y    17       0.127
  max. S    14       0.124
  mean S     1-16    0.110
       D     1-16    0.118   0.450   NO
Name=Clus_CLUG01309	SP='NO' D=0.118 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Clus_CLUG01309.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Clus_CLUG01309.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Clus_CLUG03495 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Clus_CLUG03495.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    28       0.108
  max. Y    66       0.104
  max. S    60       0.109
  mean S     1-65    0.096
       D     1-65    0.100   0.450   NO
Name=Clus_CLUG03495	SP='NO' D=0.100 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Clus_CLUG03495.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Clus_CLUG03495.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Dhan_DEHA2C16940g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Dhan_DEHA2C16940g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    17       0.125
  max. Y    17       0.114
  max. S    16       0.121
  mean S     1-16    0.101
       D     1-16    0.107   0.450   NO
Name=Dhan_DEHA2C16940g	SP='NO' D=0.107 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2C16940g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2C16940g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Dhan_DEHA2F04114g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Dhan_DEHA2F04114g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    17       0.158
  max. Y    17       0.138
  max. S     9       0.150
  mean S     1-16    0.117
       D     1-16    0.127   0.450   NO
Name=Dhan_DEHA2F04114g	SP='NO' D=0.127 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2F04114g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2F04114g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Dhan_DEHA2F04906g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Dhan_DEHA2F04906g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    44       0.107
  max. Y    44       0.106
  max. S    27       0.112
  mean S     1-43    0.097
       D     1-43    0.101   0.450   NO
Name=Dhan_DEHA2F04906g	SP='NO' D=0.101 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2F04906g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2F04906g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Dhan_DEHA2G04422g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Dhan_DEHA2G04422g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    18       0.445
  max. Y    18       0.607
  max. S     6       0.865
  mean S     1-17    0.833
       D     1-17    0.729   0.450   YES
Name=Dhan_DEHA2G04422g	SP='YES' Cleavage site between pos. 17 and 18: VIS-IS D=0.729 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2G04422g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Dhan_DEHA2G04422g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cgui_PGUG03880.1 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cgui_PGUG03880.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.196
  max. Y    24       0.182
  max. S     1       0.238
  mean S     1-23    0.183
       D     1-23    0.182   0.500   NO
Name=Cgui_PGUG03880.1	SP='NO' D=0.182 D-cutoff=0.500 Networks=SignalP-TM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG03880.1.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG03880.1.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cgui_PGUG01289.1 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cgui_PGUG01289.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    17       0.116
  max. Y    17       0.111
  max. S     9       0.128
  mean S     1-16    0.105
       D     1-16    0.108   0.450   NO
Name=Cgui_PGUG01289.1	SP='NO' D=0.108 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG01289.1.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG01289.1.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cgui_PGUG01322.1 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cgui_PGUG01322.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    20       0.115
  max. Y    11       0.105
  max. S    26       0.119
  mean S     1-10    0.098
       D     1-10    0.101   0.450   NO
Name=Cgui_PGUG01322.1	SP='NO' D=0.101 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG01322.1.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG01322.1.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cgui_PGUG02971.1 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cgui_PGUG02971.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    20       0.711
  max. Y    20       0.732
  max. S    16       0.875
  mean S     1-19    0.749
       D     1-19    0.741   0.450   YES
Name=Cgui_PGUG02971.1	SP='YES' Cleavage site between pos. 19 and 20: INA-FP D=0.741 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG02971.1.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cgui_PGUG02971.1.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ctro_CTRG05671.3 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ctro_CTRG05671.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    25       0.221
  max. Y    25       0.387
  max. S    15       0.832
  mean S     1-24    0.682
       D     1-24    0.546   0.450   YES
Name=Ctro_CTRG05671.3	SP='YES' Cleavage site between pos. 24 and 25: IMS-KN D=0.546 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05671.3.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05671.3.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ctro_CTRG05672.3 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ctro_CTRG05672.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    70       0.108
  max. Y    70       0.105
  max. S    54       0.112
  mean S     1-69    0.097
       D     1-69    0.101   0.450   NO
Name=Ctro_CTRG05672.3	SP='NO' D=0.101 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05672.3.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05672.3.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ctro_CTRG05673.3 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ctro_CTRG05673.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    54       0.118
  max. Y    54       0.106
  max. S    27       0.108
  mean S     1-53    0.097
       D     1-53    0.101   0.450   NO
Name=Ctro_CTRG05673.3	SP='NO' D=0.101 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05673.3.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05673.3.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ctro_CTRG05674.3 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ctro_CTRG05674.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    70       0.107
  max. Y    70       0.104
  max. S    50       0.115
  mean S     1-69    0.096
       D     1-69    0.100   0.450   NO
Name=Ctro_CTRG05674.3	SP='NO' D=0.100 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05674.3.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05674.3.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ctro_CTRG05833.3 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ctro_CTRG05833.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    31       0.486
  max. Y    26       0.397
  max. S     6       0.720
  mean S     1-25    0.472
       D     1-25    0.427   0.500   NO
Name=Ctro_CTRG05833.3	SP='NO' D=0.427 D-cutoff=0.500 Networks=SignalP-TM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05833.3.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG05833.3.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ctro_CTRG02612.3 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ctro_CTRG02612.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    19       0.590
  max. Y    19       0.608
  max. S    17       0.802
  mean S     1-18    0.622
       D     1-18    0.616   0.450   YES
Name=Ctro_CTRG02612.3	SP='YES' Cleavage site between pos. 18 and 19: ICS-AQ D=0.616 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG02612.3.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ctro_CTRG02612.3.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Calb_orf19.2607 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Calb_orf19_002.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    54       0.109
  max. Y    22       0.104
  max. S     9       0.118
  mean S     1-21    0.100
       D     1-21    0.102   0.450   NO
Name=Calb_orf19.2607	SP='NO' D=0.102 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.2607.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.2607.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Calb_orf19.5103 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Calb_orf19.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    18       0.156
  max. Y    18       0.115
  max. S    23       0.134
  mean S     1-17    0.086
       D     1-17    0.099   0.450   NO
Name=Calb_orf19.5103	SP='NO' D=0.099 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.5103.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.5103.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Calb_orf19.6809 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Calb_orf19_004.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    60       0.119
  max. Y    60       0.109
  max. S    26       0.112
  mean S     1-59    0.097
       D     1-59    0.103   0.450   NO
Name=Calb_orf19.6809	SP='NO' D=0.103 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.6809.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.6809.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Calb_orf19.7596 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Calb_orf19_003.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    27       0.595
  max. Y    20       0.677
  max. S    14       0.932
  mean S     1-19    0.875
       D     1-19    0.784   0.450   YES
Name=Calb_orf19.7596	SP='YES' Cleavage site between pos. 19 and 20: TLA-FP D=0.784 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.7596.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.7596.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Calb_orf19.7597 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Calb_orf19_005.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    20       0.407
  max. Y    20       0.577
  max. S     2       0.884
  mean S     1-19    0.814
       D     1-19    0.705   0.450   YES
Name=Calb_orf19.7597	SP='YES' Cleavage site between pos. 19 and 20: ATT-IS D=0.705 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.7597.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Calb_orf19.7597.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cpar_CPAG04410 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cpar_CPAG04410.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    17       0.139
  max. Y    17       0.118
  max. S     9       0.123
  mean S     1-16    0.106
       D     1-16    0.112   0.450   NO
Name=Cpar_CPAG04410	SP='NO' D=0.112 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG04410.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG04410.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cpar_CPAG02059 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cpar_CPAG02059.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    23       0.112
  max. Y    11       0.150
  max. S     1       0.263
  mean S     1-10    0.150
       D     1-10    0.150   0.450   NO
Name=Cpar_CPAG02059	SP='NO' D=0.150 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG02059.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG02059.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cpar_CPAG02227 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cpar_CPAG02227.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    22       0.142
  max. Y    22       0.118
  max. S    21       0.133
  mean S     1-21    0.098
       D     1-21    0.107   0.450   NO
Name=Cpar_CPAG02227	SP='NO' D=0.107 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG02227.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG02227.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cpar_CPAG01552 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cpar_CPAG01552.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    19       0.789
  max. Y    19       0.856
  max. S     4       0.964
  mean S     1-18    0.929
       D     1-18    0.895   0.450   YES
Name=Cpar_CPAG01552	SP='YES' Cleavage site between pos. 18 and 19: AFA-QV D=0.895 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG01552.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG01552.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Cpar_CPAG00993 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Cpar_CPAG00993.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    19       0.587
  max. Y    19       0.728
  max. S    13       0.922
  mean S     1-18    0.901
       D     1-18    0.821   0.450   YES
Name=Cpar_CPAG00993	SP='YES' Cleavage site between pos. 18 and 19: VIA-AE D=0.821 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG00993.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Cpar_CPAG00993.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Lelo_LELG00061 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Lelo_LELG00061.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    51       0.113
  max. Y    31       0.107
  max. S    43       0.126
  mean S     1-30    0.103
       D     1-30    0.105   0.450   NO
Name=Lelo_LELG00061	SP='NO' D=0.105 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG00061.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG00061.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Lelo_LELG00381 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Lelo_LELG00381.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    17       0.108
  max. Y    33       0.109
  max. S    16       0.130
  mean S     1-32    0.105
       D     1-32    0.107   0.450   NO
Name=Lelo_LELG00381	SP='NO' D=0.107 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG00381.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG00381.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Lelo_LELG03016 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Lelo_LELG03016.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    30       0.130
  max. Y    20       0.177
  max. S     9       0.342
  mean S     1-19    0.272
       D     1-19    0.228   0.450   NO
Name=Lelo_LELG03016	SP='NO' D=0.228 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG03016.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG03016.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Lelo_LELG05269 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Lelo_LELG05269.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    19       0.837
  max. Y    19       0.817
  max. S    18       0.865
  mean S     1-18    0.783
       D     1-18    0.799   0.450   YES
Name=Lelo_LELG05269	SP='YES' Cleavage site between pos. 18 and 19: VLA-DG D=0.799 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG05269.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG05269.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Lelo_LELG01999 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Lelo_LELG01999.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    21       0.833
  max. Y    21       0.743
  max. S     3       0.827
  mean S     1-20    0.650
       D     1-20    0.693   0.450   YES
Name=Lelo_LELG01999	SP='YES' Cleavage site between pos. 20 and 21: VDA-QT D=0.693 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG01999.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Lelo_LELG01999.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ylip_YALI0F03707g 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ylip_YALI0F03707g.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    56       0.116
  max. Y    56       0.107
  max. S    24       0.116
  mean S     1-55    0.099
       D     1-55    0.102   0.450   NO
Name=Ylip_YALI0F03707g	SP='NO' D=0.102 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ylip_YALI0F03707g.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ylip_YALI0F03707g.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Anid_AN4653 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Anid_AN4653.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    29       0.132
  max. Y    29       0.126
  max. S     3       0.155
  mean S     1-28    0.123
       D     1-28    0.124   0.450   NO
Name=Anid_AN4653	SP='NO' D=0.124 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Anid_AN4653.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Anid_AN4653.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Anid_AN1218 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Anid_AN1218.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.263
  max. Y    24       0.379
  max. S     3       0.773
  mean S     1-23    0.550
       D     1-23    0.472   0.450   YES
Name=Anid_AN1218	SP='YES' Cleavage site between pos. 23 and 24: VAA-KF D=0.472 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Anid_AN1218.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Anid_AN1218.gnu">gnuplot script</a>
</p><hr>
<b>&gt;Ncra_NCU02927 
</b><img src="PMU1_homologs_Ascomycetes_20171108_files/Ncra_NCU02927.png"><p>
# Measure  Position  Value    Cutoff   signal peptide?
  max. C    24       0.433
  max. Y    24       0.595
  max. S     6       0.928
  mean S     1-23    0.820
       D     1-23    0.717   0.450   YES
Name=Ncra_NCU02927	SP='YES' Cleavage site between pos. 23 and 24: ASP-EP D=0.717 D-cutoff=0.450 Networks=SignalP-noTM
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ncra_NCU02927.pred">data</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/Ncra_NCU02927.gnu">gnuplot script</a>
</p><hr>
Signal peptides: 15
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/processed.fasta">processed fasta entries</a>
# <a href="http://www.cbs.dtu.dk/services/SignalP-4.1/tmp/9781/gff.txt">gff file of processed entries</a>
</pre>
Please cite:
SignalP 4.0: discriminating signal peptides from transmembrane regions
Petersen TN., Brunak S., von Heijne G. &amp; Nielsen H.
Nature Methods, 8:785-786, 2011
<font face="ARIAL,HELVETICA">
<a href="http://www.cbs.dtu.dk/services/SignalP-4.1/output.html" target="_blank"><b>Explain</b></a> the output.  Go <a href="javascript:history.back()"><b>back</b></a>.</font>
</pre>

<hr>


</body></html>