---
title: eLife decision on phosphatase evolution paper
author: Bin He
date: '2017-11-15'
slug: elife-decision-on-phosphatase-evolution-paper
categories:
  - collaboration
  - publication
tags:
  - dennis wykoff
  - pho
---

[Link](/docs/2017-11-16-chat-with-peter-on-phosphatase-evolution) to discussion with Peter on this topic.

Our paper submitted to eLife was returned without review. Below are the critiques, followed by my thoughts

> Specifically, concerns were raised about the strength of the evidence supporting the primary conclusion that phosphatase genes specific to phosphate and thiamine starvation have arisen independently via duplication events of genes from different gene families in Ascomycetes. The evolutionary analyses of the two gene families of phosphatases discussed are restricted to just four yeast species (Candida glabrata, S. cerevisiae, S. pombe, and Kluyveromyces lactis) even though there are more than 100 yeast genomes that are publicly available. The inclusion of a genome from a filamentous fungi from Ascomycetes would be particularly important. Without examining the evolution of the two gene families (and their overlap) across the phylogeny of Ascomycetes, it is not possible to examine whether the authors' inferences about the evolution of phosphatases in fungi hold or not.

- Although we only analyzed four species and only for the _PHO5_ family, that shouldn't invalidate or prevent us from reaching our conclusion, namely the gene duplication events that gave rise to the multi-gene family in _S. pombe_, _K. lactis_ and the two _sensu stricto_ species occurred after these species have diverged from each other. 

    The gene geneaology tree for the _PHO5_ homologs strongly support our hypothesis over the alternative, i.e. the ancestor to these four species contain a superset of all the copies present in the extant species. If the alternative were true, we would expect to see some of the gene family members from one species to be more similar to a homolog from another species than to its paralogs from the same species. This is because after ancient gene duplication and selective retention, the coalescent time between any two genes in the family would be the sum of the time from present back to the speciation events plus the time the two ohnologs coalesce to the ancestor gene. Since the retension is assumed to be random, the coalescent time is not expected to follow the species division.
    
- What's the merit of adding more species to the analysis? 

    My current tree showed duplication in multiple species or clades, including the _sensu stricto_ clade, _K. lactis_, _Lanchancea_ clade, "CTG" clade and the _S. pombe_ clade. 
    
    Perhaps the point is to include species with only one member? But as I mentioned above, the gene geneology tree is sufficient to infer the relative timing of speciation vs duplication. Including intermediate (phylogenetically) species wouldn't strengthen or disprove that conclusion.
    
- Examine the overlap between the _PHO5_ and _PMU1_ families is what my new plot with signal peptide prediction is meant to achieve. I can imagine expanding this part and make it into another paper?
